scripts/o3de/ExportScripts/export_source_android.py

```diff
@@ -0,0 +1,383 @@
+                      selected_platform=expected_platform,
+                      seedlist_paths=validated_seedslist_paths,
+                      seedfile_paths=preprocessed_seedfile_paths,
+                      tools_build_path=tools_build_path,
+                      engine_centric=engine_centric,
+                      asset_bundling_path=project_path/'AssetBundling',
+                      using_installer_sdk=is_installer_sdk,
+                      tool_config=tool_config,
+                      max_bundle_size=max_bundle_size)
+
+    android_project_config= android_support.get_android_config(project_path=None)
+
+    android_project_config.set_config_value(key=android_support.SETTINGS_ASSET_BUNDLE_SUBPATH.key, value=str(asset_bundles_path).replace('\\', '/'), validate_value=False)
+    
+    android_sdk_home = pathlib.Path(android_project_config.get_value(key=android_support.SETTINGS_SDK_ROOT.key))
+    
+    if not android_sdk_home.is_dir():
+        logger.error("ANDROID_SDK_HOME is not properly configured! Please ensure SDK_ROOT is properly configured in o3de android-configure")
+        raise exp.ExportProjectError("Invalid ANDROID_SDK_HOME.")
+
+    exp.process_command([o3de_cli_script_path, 'android-generate', '-p', project_name, '-B', target_android_project_path, "--asset-mode", asset_pack_mode], 
+                cwd=default_base_path)
+
+    exp.process_command([target_android_project_path / ('gradlew'+ (".bat" if is_pc else "")), f'assemble{build_config.title()}'], cwd=target_android_project_path)
+
+    if deploy_to_device:
+        deloy_to_android_device(target_android_project_path,
+                                build_config, 
+                                project_name, 
+                                android_sdk_home,
+                                org_name,
+                                activity_name)
+
+def export_source_android_parse_args(o3de_context: exp.O3DEScriptExportContext, export_config: command_utils.O3DEConfig):
+    parser = argparse.ArgumentParser(
+                    prog=f'o3de.py export-project -es {__file__}',
+                    description="Exports a project as standalone to the desired output directory with release layout. "
+                                "In order to use this script, the engine and project must be setup and registered beforehand. ",
+                    epilog=exp.CUSTOM_CMAKE_ARG_HELP_EPILOGUE,
+                    formatter_class=argparse.RawTextHelpFormatter,
+                    add_help=False
+    )
+    parser.add_argument(exp.CUSTOM_SCRIPT_HELP_ARGUMENT,default=False,action='store_true',help='Show this help message and exit.')
+    
+    parser.add_argument('-deploy', '--deploy-to-android',default=False,action='store_true',help='At completion of build, deploy to android device.')
+
+    parser.add_argument('-actname', '--activity_name', type=str, default='',
+                        help='The name of the activity to invoke when deploying and running the APK.')
```

Comment:
Don't use `_` in the long argument. And if you were planning to defaulting the value to the project name, then that could be used here as well. (And you don't always have to use a short-form of the argument. )

```suggestion
    parser.add_argument('--activity-name', type=str, default=project_name,
                        help='The name of the activity to invoke when deploying and running the APK.')
```